src/app/Http/Requests/AccountInfoRequest.php

```diff
@@ -41,7 +41,8 @@ public function rules()
             backpack_authentication_column() => [
                 'required',
                 backpack_authentication_column() == 'email' ? 'email' : '',
-                Rule::unique($user->getTable())->ignore($user->getKey(), $user->getKeyName()),
+                Rule::unique($user->getConnectionName() ? $user->getConnectionName() . '.' . $user->getTable() : $user->getTable())
```

Comment:
```suggestion
                Rule::unique($user->getConnectionName() . '.' . $user->getTableWithPrefix())
```

Reply:
`getTableWithPrefix` instead of `getTable`

Reply:
@promatik from my tests we don't need `getTableWithPrefix` there. 

Can you confirm ? Without the prefix I got the correct table name with: ```yeah_``` as prefix.

```php
[2021-05-04 13:58:12] local.INFO: select count(*) as aggregate from `yeah_users` where `email` = ? and `id` <> ? ["vlash@sdad.com","1"] 
```

Reply:
@promatik heads up, I've just changed the commit suggestion to be accurate with what needs to be merged to finish this PR.

Reply:
@pxpm I tested once again, and I think we need the `getTableWithPrefix()`.
Are you using the prefix attribute on the connection?

Here are my tests:

`config\database.php`
```php
'mysql' => [
    'driver' => 'mysql',
    'prefix' => '',
    ...

'mysql_other' => [
    'driver' => 'mysql',
    'prefix' => 'pf_',
    ...
```

`$user->getConnectionName() . '.' . $user->getTable()`
results in `"mysql_other.users"`

`$user->getConnectionName() . '.' . $user->getTableWithPrefix()`
results in `"mysql_other.pf_users"`

Without the `getTableWithPrefix` we lose the prefix, some devs may be using it.
Let me know ✌

Reply:
Indeed @promatik , that would be the `raw` return of those functions. But in the inner-works Laravel adds the table prefix. 

Check it out, I setup 2 connections like you do, `mysql` for non prefixed and `mysql_2` for prefixed connection. `yeah_` is my prefix.

All my tables in database don't have the prefix, so to create them, I change default laravel connection to `mysql_2` and run `php artisan migrate` again, I will have `users` and `yeah_users` table in database. (change back the default connection to `mysql`)

In the user model, I changed default connection to use the prefixed one:

`$connection = "mysql_2";` 

Without using the `getTableWithPrefix()`, when I try to change the email on the account info (where the request is triggered), I get the expected `yeah_users` prefixed table call:

```php
[2021-05-05 11:04:54] local.INFO: select count(*) as aggregate from `yeah_users` where `email` = ? and `id` <> ? ["djamba@lola.com","1"] 
``` 
If I remove the connection from the model, so it will use the default without prefix, i get the expected call too:
```php
[2021-05-05 11:10:38] local.INFO: select count(*) as aggregate from `users` where `email` = ? and `id` <> ? ["djaasdasda@lola.com","1"] 
```
To log the queries I use this function in AppServiceProvider:

```php
DB::listen(function($query) {
            Log::info(
                $query->sql,
                $query->bindings,
                $query->time
            );
        });
```

Let me know if I am doing something wrong in the testing process that is leading me to wrong results.

Best,
Pedro

Reply:
@pxpm you are right 👌 that is indeed the raw query, laravel does the job in the background 🙌
Since you've tested, and me too, I'll commit this and move to ready to merge.